src/components/Headers/DefaultHeader/DefaultHeader.tsx

```diff
@@ -14,46 +14,35 @@ const DefaultHeader = () => {
 
   const router = useRouter()
 
-  if (status === 'authenticated') {
-    return (
-      <header className={styles.wrapper}>
-        <Link href="/" className={styles.logo}>
-          <img src={logo.src} alt="Logo" />
-          <div className={styles.title}>Good Dev Hunting</div>
-        </Link>
-        <div className={styles.actions}>
-          <div className={styles.frameLogin}>
-            <Button
-              onClick={() => router.push('/my-profile')}
-              variant={'primary'}
-            >
-              {' '}
-              My profile{' '}
-            </Button>
-          </div>
-        </div>
-      </header>
-    )
-  }
-
   return (
     <header className={styles.wrapper}>
       <Link href="/" className={styles.logo}>
         <img src={logo.src} alt="Logo" />
         <div className={styles.title}>Good Dev Hunting</div>
       </Link>
-
-      <div className={styles.frameButtons}>
-        <Button onClick={() => signIn('github')} variant={'secondary'}>
-          Login
-          <GithubIcon />
-        </Button>
-        <Button onClick={() => signIn()} variant={'primary'}>
-          Create profile
-          <AddIcon />
-        </Button>
-      </div>
+      {status === 'authenticated' ? (
+        <div className={styles.frameButtons}>
+          <Button
+            onClick={() => router.push('/my-profile')}
+            variant={'primary'}
```

Comment:
You should use routes from src/utils

Reply:
**nitpick**: What wawrozn means is: ` onClick={() => router.push(AppRoutes.MyProfile)}`